### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by zivy**

### CODE REVIEW

**Comment by blowekamp** [comment]
These changes look good. Can these new pages be under an "about" tab? I think you can create an about subdirectory and then include an index.rst file in it. Something similar that was done for the examples: https://github.com/SimpleITK/SimpleITK/blob/master/Examples/index.rst

**Comment by zivy** [comment]
Sure, will modify and force push.

File: docs/source/history.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+SimpleITK was created as part of a concerted effort to simplify the use of the Insight toolkit, making it 
+more accessible to a wider audience. 
+
+The initial funding for the toolkit was provided by the United States National Library of Medicine (NLM) 
+under the American Recovery and Reinvestment Act (ARRA), with the initial version of the toolkit 
+developed as a collaboration between The Mayo Clinic, Kitware Inc, The University of Iowa and NLM's 
+intramural research program. The first major release of the toolkit was announced in April-May 2017.
+
+Between 2013 and 2019, development was primarily carried out as part of the NLM intramural research
```
Comment (blowekamp) [inline docs/source/history.rst]: Please change this to "supported by"

File: docs/source/index.rst [hunk 2]
```diff
@@ -26,7 +26,7 @@ SimpleITK are currently available for all three major operating systems
    faq
    FilterTemplates
    link_examples
-
+   about
```
[the remark this answers is not in the document]
Comment (zivy) [reply docs/source/index.rst]: Yes, moved to top.

File: docs/source/about.rst [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.. _lbl_about:
+
+About
```
[the remark this answers is not in the document]
Comment (zivy) [reply docs/source/about.rst]: Left it as "About" as everything on the site is with respect to SimpleITK.

File: docs/source/history.rst (hunk 1 above)
Comment (zivy) [reply docs/source/history.rst]: Changed.

**Pull request merged**